.github/workflows/trunk.yml

```diff
@@ -131,6 +131,24 @@ jobs:
         # Test selective build
         PYTHON_EXECUTABLE=python bash examples/portable/scripts/test_demo_backend_delegation.sh "${BUILD_TOOL}"
 
+  test-arm-backend-delegation:
+    name: test-arm-backend-delegation
+    uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
+    with:
+      runner: linux.2xlarge
+      docker-image: executorch-ubuntu-22.04-clang12-arm-sdk
```

Comment:
The name here should be `executorch-ubuntu-22.04-arm-sdk` I think.  Once you set the correct name, the failed jobs here https://github.com/pytorch/executorch/actions/runs/6712861373/job/18243312911 should use the correct Docker image

Reply:
Oops